srcpkgs/atheme/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'atheme'
+pkgname=atheme
+version=7.2.12
+revision=1
+build_style=gnu-configure
+configure_args="--disable-nls --enable-fhs-paths"
+hostmakedepends="pkg-config gettext"
+makedepends="libmowgli-devel"
+short_desc="Atheme IRC Services"
+maintainer="Luca Matei Pintilie <luca@lucamatei.com>"
+license="ISC"
+homepage="https://atheme.github.io/"
+changelog="https://github.com/atheme/atheme/releases"
+distfiles="https://github.com/atheme/atheme/releases/download/v$version/atheme-services-v$version.tar.xz"
+checksum=600d5ba0e25d80307818021fd9b85e49b07f245932b3c8e04e16fe312dd60fba
+conflicts="ecdsautils lifelines" # /usr/bin/ecdsakeygen and /usr/bin/dbverify
```

Comment:
would it make sense to change the names of these to `atheme-ecdsakeygen` and `atheme-dbverify`?